### ISSUE TITLE
login error!

### ISSUE THREAD

**Issue by whichdew** [apache/pulsar-manager#471]
after deploy pulsar with helm, i can't login pulsar-manager with default credencials! how to solve it?

**Comment by whichdew** [comment]
```shell
CSRF_TOKEN=$(curl http://backend-service:7750/pulsar-manager/csrf-token)
curl \
    -H "X-XSRF-TOKEN: $CSRF_TOKEN" \
    -H "Cookie: XSRF-TOKEN=$CSRF_TOKEN;" \
    -H 'Content-Type: application/json' \
    -X PUT http://backend-service:7750/pulsar-manager/users/superuser \
    -d '{"name": "admin", "password": "apachepulsar", "description": "test", "email": "username@test.org"}'
```

**Comment by haiboself** [comment]
> ```shell
> CSRF_TOKEN=$(curl http://backend-service:7750/pulsar-manager/csrf-token)
> curl \
>     -H "X-XSRF-TOKEN: $CSRF_TOKEN" \
>     -H "Cookie: XSRF-TOKEN=$CSRF_TOKEN;" \
>     -H 'Content-Type: application/json' \
>     -X PUT http://backend-service:7750/pulsar-manager/users/superuser \
>     -d '{"name": "admin", "password": "apachepulsar", "description": "test", "email": "username@test.org"}'
> ```

same issue, the curl return "Empty reply from server" as response, Do you solve this problem in the final?

**Comment by xdewx** [comment]
> > ```shell
> > CSRF_TOKEN=$(curl http://backend-service:7750/pulsar-manager/csrf-token)
> > curl \
> >     -H "X-XSRF-TOKEN: $CSRF_TOKEN" \
> >     -H "Cookie: XSRF-TOKEN=$CSRF_TOKEN;" \
> >     -H 'Content-Type: application/json' \
> >     -X PUT http://backend-service:7750/pulsar-manager/users/superuser \
> >     -d '{"name": "admin", "password": "apachepulsar", "description": "test", "email": "username@test.org"}'
> > ```
> 
> same issue, the curl return "Empty reply from server" as response, Do you solve this problem in the final?

I solved it. make sure:
1. pod & service all is up
2. follow instruction in README

**Comment by haiboself** [comment]
> > > ```shell
> > > CSRF_TOKEN=$(curl http://backend-service:7750/pulsar-manager/csrf-token)
> > > curl \
> > >     -H "X-XSRF-TOKEN: $CSRF_TOKEN" \
> > >     -H "Cookie: XSRF-TOKEN=$CSRF_TOKEN;" \
> > >     -H 'Content-Type: application/json' \
> > >     -X PUT http://backend-service:7750/pulsar-manager/users/superuser \
> > >     -d '{"name": "admin", "password": "apachepulsar", "description": "test", "email": "username@test.org"}'
> > > ```
> > 
> > 
> > same issue, the curl return "Empty reply from server" as response, Do you solve this problem in the final?
> 
> I solved it. make sure:
> 
> 1. pod & service all is up
> 2. follow instruction in README

Thanks, Finally, I solved the problem by installing the v0.3.0 version rather than v0.2.0.

**Comment by saurabh21316** [comment]
@haiboself could you please help how did you get it resolved?
